cns/NetworkContainerContract.go

```diff
@@ -98,6 +100,7 @@ type KubernetesPodInfo struct {
 }
 
 // GetOrchestratorContext will return the orchestratorcontext as a string
+// TODO - should use a hashed name or can this be PODUid?
```

Comment:
I think this should be Pod resource ID (GUID) because a Pod can be deleted and recreated with the same name, and we want to know that it's different

Reply:
Discussed offline, this value is passed to CNI by kubelet. We should discuss this in our syncup that kubelet needs to call cni for first pod deletion before calling add for another.